### ISSUE TITLE
How to embody this characteristic: True Determinism

### ISSUE THREAD

**Issue by gongel** [mosaicml/streaming#393]
Hi all, could you show some codes to show "True Determinism"?
When “Load a checkpoint trained on 64 GPUs and debug on 8 GPUs with reproducibility”, we should set gradient_accumulation_steps to control same global batch size?

**Comment by karan6181** [comment]
As far as streaming Dataset parameter goes, ensure you have same `shuffle_seed` and `num_canonical_nodes`. With regards to the training hyperparams, ensure global batch size remains the same between your runs.

**Comment by gongel** [comment]
Thx, got it
